AsyncDisplayKit/Layout/ASLayoutElementExtensibility.h

```diff
@@ -30,5 +34,80 @@
 - (void)setLayoutOptionExtensionEdgeInsets:(UIEdgeInsets)value atIndex:(int)idx;
 - (UIEdgeInsets)layoutOptionExtensionEdgeInsetsAtIndex:(int)idx;
 
-
 @end
+
+#pragma mark - Dynamic Properties
+
+/**
+ * Unbox NSNumber based on the type
+ */
+#define ASDK_UNBOX_NUMBER(NUMBER, PROPERTY_TYPE) \
```

Comment:
I think for now this will be fine and we can re-consider if we ran into issues. If a not supported data type is set we will assert for now.